---
event_type: IssueCommentEvent
avatar: "https://avatars.githubusercontent.com/u/4656391?"
user: wdconinc
date: 2024-04-21
repo_name: spack/spack
html_url: https://github.com/spack/spack/issues/31280
repo_url: https://github.com/spack/spack
---

<a href='https://github.com/wdconinc' target='_blank'>wdconinc</a> commented on issue <a href='https://github.com/spack/spack/issues/31280' target='_blank'>spack/spack#31280</a>.

<small>This issue appears when (because) /usr/local already exists, is a directory, and isn't empty. In that case, spack won't remove it. Exception [IsADirectoryError](https://docs.python.org/3.10/library/exceptions.html#IsADirectoryError) is raised in https://github.com/spack/spack/blob/7c1a3094536730be698809daf2c3df267dd8a91b/lib/spack/llnl/util/filesystem.py#L226...</small>

<a href='https://github.com/spack/spack/issues/31280' target='_blank'>View Comment</a>